src/gen/models/index.ts

```diff
@@ -7078,6 +6976,22 @@ export interface ReviewQueueItemResponse {
   moderation_payload?: ModerationPayload;
 }
 
+export interface ReviewQueueItemUpdatedEvent {
+  created_at: Date;
+
+  custom: Record<string, any>;
+
+  type: string;
+
+  received_at?: Date;
+
+  flags?: Flag2Response[];
```

Comment:
@viktorapo808 Flag2Response looks very ugly. Should we change the struct name to something better - `ModerationFlagResponse`